src/plots/cartesian/layout_attributes.js

```diff
@@ -593,6 +593,15 @@ module.exports = {
             '*%H~%M~%S.%2f* would display *09~15~23.46*'
         ].join(' ')
     },
+    hovertitle: {
+        valType: 'boolean',
```

Comment:
That's not what I had in mind.

I was thinking of making `hovertitle` a string that defaults to the current behavior (in v1.x, and potentially default to the axis `title` value in v2.x).

Reply:
The `hovertitle` is now a `string` instead of a `boolean`. 
Other axis types have `_hovertitle` attribute. Wondering if we should change them to `hovertitle`?

Reply:
Sounds like those `_hovertitle` are now obsolete. Would you mind :hocho: ing them? Thanks!